### PR TITLE
Bugs/various bug fixed

### PR DESCRIPTION
**Pull request by julia-thea**
This PR addresses the following:

- Passing an actual product name instead of hard coded one
- Review count is now updated immediately upon review submission
- Fixed the bug where clicking on sunglasses crashes the app
- Updated app title

@acdavitt @dylanreid7 @ChrisRPeterson

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>MyApp</title>
+    <title>J. CAD</title>
```
Comment (dylanreid7) [inline dist/index.html:4]: Wooo!  I like it :)

File: client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const AddReviewDialog = ({ productId }) => {
     axios
       .post(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/reviews`, formDataStorage.data)
       .then(res => {
-      console.log(res)
+        handleSubmitReview(true);
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx:41]: It seems to me that handleSubmitReview doesn't take any arguments, but I could be missing something...

File: client/components/RatingsAndReviews/Rating/Factors.jsx [hunk 3]
```diff
@@ -53,6 +53,10 @@ const marks = [
 export default function CustomizedSlider({ characteristics }) {
   const classes = useStyles();
 
+  if (characteristics.Quality) {
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Rating/Factors.jsx:56]: Should this be if (!characteristics.Quality) ?

File: dist/index.html (hunk 1 above)
Comment (ChrisRPeterson) [reply dist/index.html:4]: 10/10

File: client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx (hunk 2 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx:41]: Good catch. I realized later that I don't need to pass ```true``` as a param and can just update state inside the event handler, but forgot to correct the function invocation here.

File: client/components/RatingsAndReviews/Rating/Factors.jsx (hunk 3 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Rating/Factors.jsx]: Pushed the fix

**Comment by julia-thea** [comment]
@dylanreid7 addressed your comments, please take another look

**Review by dylanreid7 — APPROVED** [review]

**Pull request merged**